### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by raphox** [vstirbu/ZeroConf#23]
I do not want to belittle the work of the owner of this repository. But by searching I located the repository https://github.com/becvert/cordova-plugin-zeroconf, which seems to be more complete.

**Comment by vstirbu** [comment]
yes, I'm aware that there are quite many forks of this project. somehow it seems that there is not enough gravity to pull all the efforts together.
